src/Raven.Server/Documents/Handlers/CountersHandler.cs

```diff
@@ -79,6 +80,22 @@ public void Add(string id, CounterOperation op)
                 }
             }
 
+            public void Gather(DocumentsOperationContext context, string id, StringValues counters)
```

Comment:
Why this roundabout way of doing this? Instead of creating a command, then executing it, let's do this directly.

Reply:
Just for consistency.. We use includes. Gather. Will remove this and build CountersBatch object directly, then pass to command ctr and execute

Reply:
@ayende this actually depends, if we are going to use this in more than 1 place, then it would be better to wrap it in a command as we do with regular includes, so the code would be more reusable and maintainable. And I guess we will use it everywhere we use regular includes?

Reply:
Yes, having all of the include behavior in one location is better, now that we are doing more of that (and we'll continue to do more)